### PR TITLE
feature(export) add simple ergonode csv export

### PR DESCRIPTION
**Pull request by rprzedzik**
# Description

Add simple ergonode csv export channel

Issue # 650
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] Unit Tests
- [x] Behat Test

# Checklist:
- [x] I have read the contribution requirements and fulfill them.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: module/channel/src/Application/Command/ExportChannelCommand.php [hunk 1]
```diff
@@ -75,27 +63,16 @@ public function configure(): void
...
         $output->writeln(sprintf('Processing products witch channel <comment>%s</comment>', $channel->getName()));
 
-        foreach ($ids as $id) {
-            $progressBar->advance();
-            $progressBar->setMessage($id);
-            $productId = new ProductId($id);
-            $command = new ExportProductChannelDomainCommand($channelId, $productId);
-            $this->commandBus->dispatch($command);
-        }
+        $command = new \Ergonode\Channel\Domain\Command\ExportChannelCommand(ExportId::generate(), $channelId);
```
Comment (wfajczyk) [inline module/channel/src/Application/Command/ExportChannelCommand.php:74]: Why are you using longer path ? 
Please change to alias.

File: module/exporter-file/src/Infrastructure/Processor/ProductProcessor.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        $result->set('_sku', $product->getSku()->getValue());
+        $result->set('_type', $product->getType());
+        $result->set('_template', $product->getTemplateId()->getValue());
+        foreach ($attributes as $attributeId => $code) {
+            $code = new AttributeCode($code);
+            if ($product->hasAttribute($code)) {
+                $attribute = $this->attributeRepository->load(new AttributeId($attributeId));
+                Assert::isInstanceOf($attribute, AbstractAttribute::class);
+                $value = $product->getAttribute($code);
+                $calculatedValue = $this->calculator->calculate($attribute, $value, $language);
```
Comment (wfajczyk) [inline module/exporter-file/src/Infrastructure/Processor/ProductProcessor.php:108]: What about multimedia and select field? This is where the id will be set ?

File: module/exporter-file/src/Infrastructure/Writer/XmlWriter.php [hunk 3]
```diff
@@ -105,47 +64,26 @@ public function write(AbstractProduct $product, array $languages, array $attribu
...
-     * @return string|null
-     */
-    private function getValue(AbstractProduct $product, AttributeCode $code, Language $language): ?string
-    {
-        if ($product->hasAttribute($code)) {
-            $attributeId = AttributeId::fromKey($code->getValue());
-            $attribute = $this->repository->load($attributeId);
-            Assert::notNull($attribute);
-            $value = $product->getAttribute($code);
+        fputcsv($buffer, $data);
```
Comment (wfajczyk) [inline module/exporter-file/src/Infrastructure/Writer/XmlWriter.php:81]: Are You sure You are use `fputcsv` in XML writter

File: module/exporter-file/src/Application/Model/ExporterFileConfigurationModel.php [hunk 4]
```diff
@@ -23,6 +23,8 @@ class ExporterFileConfigurationModel
      */
     public ?string $name = null;
 
+    public array $languages = [];
```
[the remark this answers is not in the document]
Comment (rprzedzik) [reply module/exporter-file/src/Application/Model/ExporterFileConfigurationModel.php:26]: Validation is seta as choices in ChoceFormType

File: module/exporter-file/src/Infrastructure/Processor/ProductProcessor.php (hunk 2 above)
Comment (rprzedzik) [reply module/exporter-file/src/Infrastructure/Processor/ProductProcessor.php:108]: yes

**Pull request merged**